sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/interface.scala

```diff
@@ -419,7 +419,7 @@ case class CatalogTable(
     map ++= storage.toLinkedHashMap
     if (tracksPartitionsInCatalog) map.put("Partition Provider", "Catalog")
     if (partitionColumnNames.nonEmpty) map.put("Partition Columns", partitionColumns)
-    if (schema.nonEmpty) map.put("Schema", schema.treeString)
+    if (schema.nonEmpty) map.put("Schema", schema.catalogString)
```

Comment:
How about removing this line. It seems other DBs does not contains Schema.

Reply:
It's no harm to keep it